recipes/eigen/bld.bat

```diff
@@ -0,0 +1,6 @@
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target INSTALL --config Release
```

Comment:
I believe it just sets some defines and then copies the files. It depends on if you want to build and run the tests - they obviously require building.